api/src/middleware/authenticate.test.ts

```diff
@@ -266,3 +276,79 @@ test('Sets accountability to user information when static token is used', async
 	expect(req.accountability).toEqual(expectedAccountability);
 	expect(next).toHaveBeenCalledTimes(1);
 });
+
+test('Invalid session token clears the cookie', async () => {
```

Comment:
To make clear that we still expect the response to fail when clearing the cookie

```suggestion
test('Invalid session token responds with error and clears the cookie', async () => {
```